.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "MP-SPDZ"]
-	path = MP-SPDZ
+	path = mp-spdz-rs/include/MP-SPDZ
+	url = https://github.com/renegade-fi/MP-SPDZ.git
+[submodule "mp-spdz-rs/src/include/MP-SPDZ"]
+	path = mp-spdz-rs/src/include/MP-SPDZ
```

Comment:
Is this a duplicate? Looks like we create a submodule for the same repo at one level deeper in the directory